File: WebClient SDK/src/AutologoutDialog/registrator.ts
import { ExtensionContainer, ExtensionStage } from 'tessa/extensions';
import {  AutoLogoutDialog, LogExtension } from './AutoLogout';

ExtensionContainer.instance.registerExtension({
  extension: AutoLogoutDialog,
  stage: ExtensionStage.AfterPlatform,
});
ExtensionContainer.instance.registerExtension({
  extension: LogExtension,
  stage: ExtensionStage.AfterPlatform,
});

File: Extensions/Tessa.Extensions.Default.Server/Workflow/Registrator.cs
using Tessa.Cards;
using Tessa.Cards.Extensions;
using Tessa.Extensions.Default.Shared;

namespace Tessa.Extensions.Default.Server.Workflow
{
    [Registrator]
    public class Registrator : RegistratorBase
    {
        #region Base Overrides

        /// <inheritdoc/>
        public override void RegisterExtensions(IExtensionContainer extensionContainer)
        {
            extensionContainer
                .RegisterExtension<ICardStoreTaskExtension, KrClearWasteInAdditionalApprovalStoreTaskExtension>(x => x
                    .WithOrder(ExtensionStage.AfterPlatform, 998)
                    .WithSingleton()
                    .WhenTaskTypes(DefaultTaskTypes.KrAdditionalApprovalTypeID))
                .RegisterExtension<ICardStoreTaskExtension, KrClearWasteInApprovalStoreTaskExtension>(x => x
                    .WithOrder(ExtensionStage.AfterPlatform, 999)
                    .WithSingleton()
                    .WhenTaskTypes(DefaultTaskTypes.KrApproveTypeID, DefaultTaskTypes.KrSigningTypeID, DefaultTaskTypes.KrAdditionalApprovalTypeID))
                ;
        }

        #endregion
    }
}
